lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,34 @@
 local util = require 'lspconfig.util'
 
+local function activate_env()
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_client = util.get_active_client_by_name(bufnr, 'julials')
+  if not julials_client then
+    vim.notify(
+      'method julia/activateenvironment is not supported by any servers active on the current buffer',
+      vim.log.levels.WARN
+    )
+    return
+  end
+  local julia_project_files = { 'Project.toml', 'JuliaProject.toml' }
+  local depot_envs = vim.fs.normalize(vim.env.JULIA_DEPOT_PATH or vim.fn.expand '~/.julia') .. '/environments'
```

Comment:
This kind of stuff (reading env var JULIA_DEPOT_PATH) is a bad sign. How are users going to know about that? That's a hidden assumption in the impl of this command. Should be documented at least, in the `docs` field below.

Reply:
Ok, I can document it in the description. The environment variable `JULIA_DEPOT_PATH` is a special environment variable dictated by the programming language itself.

https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH

Reply:
Ok, I added this documentation in the latest commit revision. Let me know if it sounds alright.